components/speedreader/BUILD.gn

```diff
@@ -0,0 +1,29 @@
+import("//build/buildflag_header.gni")
```

Comment:
this should be in components/speedreader/browser except for maybe switches in common?